nca/FWRules/ConnectivityGraph.py

```diff
@@ -384,6 +448,39 @@ def get_connectivity_dot_format_str(self, connectivity_restriction=None):
                 dot_graph.add_edge(src_name=edge[0][0], dst_name=edge[1][0], label=conn_str, is_dir=False)
         return dot_graph.to_str()
 
+    def convert_to_tcp_like_properties(self, peer_container):
```

Comment:
update renaming of `tcp_props` to `conn_props`

Reply:
Updated

Reply:
this function name is still `convert_to_tcp_like_properties`

Reply:
this function is not in use?

Reply:
Right, it's not in use. Removed